acceptance/testdata/restorer/container/layers/cacher_buildpack/cached-layer.toml

```diff
@@ -1,3 +1,3 @@
 build = false
 launch = false
-cache = true
+cache = true
```

Comment:
Do we need a new line at the end of the file?

Reply:
I asked about this before, and I think at the time we said no because the analyzer wasn't writing the file with a newline, so it's more realistic this way. But I noticed that the "layer metadata restorer" is now writing these files with a newline (I had to update some expectations in restorer_test.go) so maybe yes is the way to go? @jabrown85 WDYT

Reply:
I always prefer trailing newlines for diffing reasons. But I understand this file is generated by a machine, not a person. 

I don't recall changing anything about newlines though, did this change when we rewrite the `layer.toml` values with `false` and toml table maybe?

Reply:
I actually misread this as you wanted to _remove_ the line initially. I didn't realize they were missing, I added a few in ed128ba